acceptance/analyzer_test.go

```diff
@@ -1015,6 +1051,170 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 				assertAnalyzedMetadata(t, filepath.Join(copyDir, "some-other-layers", "analyzed.toml")) // analyzed.toml is written at the provided -layers directory: /some-other-layers
 			})
 		})
+
+		when("validating stack", func() {
+			it.Before(func() {
+				h.SkipIf(t, api.MustParse(platformAPI).Compare(api.MustParse("0.7")) < 0, "Platform API < 0.7 does not validate stack")
```

Comment:
This is interesting - technically earlier platform specs did claim to validate the stack id, it just wasn't done in practice. But I guess in practice we're also trying not to change behavior for previous platform apis...